topics/statistics/tutorials/RNN/slides.html

```diff
@@ -0,0 +1,223 @@
+- 2D array. Rows represent documents. Columns represent words in vocabulary
+	- All unique words in all documents
+- If a word not present in a document, we have a zero at row and column entry
+- If a word is present in a document, we have a one at row and colum entry
+	- Or, we could use the word count or frequency
+
+---
+
+# Bag of words (BoW)
+
+- Document 1: Magic passed the basketball to Kareem
+- Document 2: Lebron stole the basketball from Curry
+
+![Table showing a bag-of-words representation of sample documents]({{site.baseurl}}/topics/statistics/images/BoW.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+- BoW is simple, but does not consider rarity of words across documents
+	- Important for document classification
+
+---
+
+# Term frequency inverse document frequency (TF-IDF)
+
+- If you don’t care about the order of the words in a document
+- Similar to BoW, we have an entry for each document-word pair
+- Entry is product of
+	- Term frequency, frequency of a word in a document, and
+	- Inverse document frequency, total number of documents divided by number of documents that have word
+		- Usually use logarithm of the IDF
+- TF-IDF takes into account rarity of a word across documents
+
+---
+
+# One-hot encoding (OHE)
+
+- Technique to convert categorical variables such as words into a vector
+- Suppose our vocabulary has 3 words: orange, apple, banana
+- Each word is represented by a vector of size 3
+
+![Mathematical vectors representing one-hot-encoding representation of words orange, apple, and banana]({{site.baseurl}}/topics/statistics/images/OHE.gif) <!-- https://pixy.org/3013900/ CC0 license-->
+
+- OHE problems
+	- For very large vocabulary sizes requires tremendous amount of storage
+	- Also, no concept of word similarity
+
+---
+
+# Word2Vec
+
+- Each word represented as an *n* dimensional vector
+	- *n* much smaller than vocabulary size
+- Words that have similar meanings are close in vector space
+- Words considered similar if they co-occur often in documents
+- Two Word2Vec architectures
+	- Continuous BoW
+		- predicts probability of a word given the surrounding words
+	- Continuous skip-gram
+		- given a word predicts probability of the surrounding words
+
+---
+
+# Sentiment analysis
+
+- Sentiment classification of IMDB movie reviews with RNN
+- Train RNN using IMDB movie reviews
+- Goal is to learn a model such that given a review we predict whether review is positive/negative
+- We evaluate the trained RNN on test dataset and plot confusion matrix
+
+---
+
+# For references, please see tutorial's References section
+
+---
+
+- Galaxy Training Materials ([training.galaxyproject.org](https://training.galaxyproject.org))
+
+![]({{site.baseurl}}/topics/introduction/images/gtn_stats.png)
+
+???
+
+- If you would like to learn more about Galaxy, there are a large number of tutorials available.
+- These tutorials cover a wide range of scientific domains.
+
+---
+
+# Getting Help
```

Comment:
I like that you added these final slides! I think we should make them part of the framework so that they show up as an appendix/resources slide at the end of every GTN slide deck, what do you think? (then you can remove them here again for now)

Reply:
Yes, thats a good idea. In the mean time, should I remove them from my slides?

Reply:
yes, please remove them here and I will add them to the template next week orso :+1:

Reply:
Ok, changes pushed